packaging/installer/README.md

```diff
@@ -107,28 +107,56 @@ troubleshooting with Netdata.
 <InstallRegexLink>
   <InstallBoxRegexLink
     to="[](https://github.com/netdata/netdata/blob/master/packaging/docker/README.md)"
-    os="Run with Docker" 
+    os="Run with Docker"
     svg="docker" />
   <InstallBoxRegexLink
     to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kubernetes.md)"
-    os="Deploy on Kubernetes" 
+    os="Deploy on Kubernetes"
     svg="kubernetes" />
    <InstallBoxRegexLink
     to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/macos.md)"
-    os="Install on macOS" 
+    os="Install on macOS"
     svg="macos" />
   <InstallBoxRegexLink
     to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/manual.md)"
-    os="Linux from Git" 
+    os="Linux from Git"
     svg="linux" />
   <InstallBoxRegexLink
     to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/source.md)"
     os="Linux from source"
     svg="linux" />
   <InstallBoxRegexLink
-    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/offline.md)" 
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/offline.md)"
     os="Linux for offline nodes"
     svg="linux" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/ansible.md)"
+    os="Deploy on Ansible"
+    svg="ansible" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/cloud-providers.md)"
+    os="Deploy on cloud providers"
+    svg="cloud" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/freebsd.md)"
+    os="Install on FreeBSD"
+    svg="freebsd" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/freenas.md)"
+    os="Install on FreeNAS"
+    svg="freenas" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/packages.md)"
+    os="Install with native packages"  
+    svg="package" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/pfsense.md)"
+    os="Install on pfSense"
+    svg="pfsense" />
+  <InstallBoxRegexLink
+    to="[](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/synology.md)"
+    os="Install on Synology"
+    svg="synology" />
```

Comment:
Do we want to keep methods that we don't actually support/don't even know if they work (I bet they don't)? cc @netdata/agent-sre 

I guess there were missing for a reason.

Reply:
Well, that's actually a good question, if we don't support them, then we need to mention it in their markdown files and obviously not include them, or something similar
@cakrit

Reply:
Everything that exists under packaging/installer/methods is supported, either by Netdata or someone else.
If something is no longer supported, it needs to be removed from the repo. @Ferroin @tkatsoulas plz verify the list and make any necessary changes to the files in the repo and to this document.